upskirtref_test.go

```diff
@@ -49,9 +50,14 @@ func doTestsReference(t *testing.T, files []string, flag int) {
 			continue
 		}
 		expected := string(expectedBytes)
+		expected = strings.Replace(expected, "\r\n", "\n", -1)
+		expected = strings.Replace(expected, "\r", "\n", -1)
 
 		// fmt.Fprintf(os.Stderr, "processing %s ...", filename)
 		actual := string(runMarkdownReference(input, flag))
+		actual = strings.Replace(actual, "\r\n", "\n", -1)
+		actual = strings.Replace(actual, "\r", "\n", -1)
```

Comment:
0d9aa1ca7ef663f9e67a7031a31370f52e2ac9a6 removed one instance, but this other one remains.